### DISCUSSION TITLE
What are types of plugin in semantic kernel?

### DISCUSSION THREAD

**Discussion by rakshahulle** [microsoft/semantic-kernel#9803] [Q&A] [answered]
Are Native Plugin and Semantic Plugin types of SK according to the latest release of SK? If not then what are types?

**Comment by sophialagerkranspandey** [comment, marked as answer]
This section will give a good overview of what you're looking for: https://learn.microsoft.com/en-us/semantic-kernel/concepts/plugins/?pivots=programming-language-csharp
